app/tests/spec/views/complete_account_unlock.js

```diff
@@ -0,0 +1,219 @@
+      sinon.stub(fxaClient, 'completeAccountUnlock', function () {
+        if (accountUnlockError) {
+          return p.reject(accountUnlockError);
+        } else {
+          return p();
+        }
+      });
+
+      account = user.initAccount({
+        sessionToken: 'foo',
+        email: 'a@a.com',
+        uid: validUid
+      });
+
+      initView();
+    });
+
+    afterEach(function () {
+      metrics.destroy();
+
+      view.remove();
+      view.destroy();
+
+      view = windowMock = metrics = null;
+    });
+
+    describe('render', function () {
+      it('shows an error if uid is not available on the URL', function () {
+        return testShowsDamagedScreen(Url.objToSearchString({
+          code: validCode
+        }))
+        .then(function () {
+          assert.isFalse(view.fxaClient.completeAccountUnlock.called);
+        });
+      });
+
+      it('shows an error uid is invalid', function () {
+        return testShowsDamagedScreen(Url.objToSearchString({
+          uid: invalidUid,
+          code: validCode
+        }))
+        .then(function () {
+          assert.isFalse(view.fxaClient.completeAccountUnlock.called);
+        });
+      });
+
+      it('shows an error if code is not available on the URL', function () {
+        return testShowsDamagedScreen(Url.objToSearchString({
+          uid: validUid
+        }))
+        .then(function () {
+          assert.isFalse(view.fxaClient.completeAccountUnlock.called);
+        });
+      });
+
+      it('shows an error code is invalid', function () {
+        return testShowsDamagedScreen(Url.objToSearchString({
+          uid: validUid,
+          code: invalidCode
+        }))
+        .then(function () {
+          assert.isFalse(view.fxaClient.completeAccountUnlock.called);
+        });
+      });
+
+      it('INVALID_PARAMETER error displays the verification link damaged screen', function () {
+        accountUnlockError = AuthErrors.toError('INVALID_PARAMETER', 'code');
+        return testShowsDamagedScreen()
+          .then(function () {
+            assert.isTrue(view.fxaClient.completeAccountUnlock.called);
+          });
+      });
+
+      it('UNKNOWN_ACCOUNT error displays the verification link expired screen', function () {
+        accountUnlockError = AuthErrors.toError(
+            'UNKNOWN_ACCOUNT', 'who are you?');
+        return testShowsExpiredScreen()
+          .then(function () {
+            assert.isTrue(view.fxaClient.completeAccountUnlock.called);
+          });
+      });
+
+      it('INVALID_VERIFICATION_CODE error displays the verification link damaged screen', function () {
+        accountUnlockError = AuthErrors.toError(
+            'INVALID_VERIFICATION_CODE', 'this isn\'t a lottery');
+        return testShowsDamagedScreen()
+          .then(function () {
+            assert.isTrue(view.fxaClient.completeAccountUnlock.called);
+          });
+      });
+
+      it('all other server errors are displayed', function () {
```

Comment:
suggest "displayed verbatim" or similar to make the intended semantics a little clearer.